src/core/src/OmniRasterOverlay.cpp

```diff
@@ -38,6 +39,32 @@ double OmniRasterOverlay::getAlpha() const {
     return static_cast<double>(alpha);
 }
 
+float OmniRasterOverlay::getMaximumScreenSpaceError() const {
+    const auto cesiumRasterOverlay = UsdUtil::getCesiumRasterOverlay(_pContext->getUsdStage(), _path);
+    float value;
+    cesiumRasterOverlay.GetMaximumScreenSpaceErrorAttr().Get(&value);
+    return static_cast<float>(value);
+}
+
+int OmniRasterOverlay::getMaximumTextureSize() const {
+    const auto cesiumRasterOverlay = UsdUtil::getCesiumRasterOverlay(_pContext->getUsdStage(), _path);
+    int value;
+    cesiumRasterOverlay.GetMaximumTextureSizeAttr().Get(&value);
+    return static_cast<int>(value);
+}
+int OmniRasterOverlay::getMaximumSimultaneousTileLoads() const {
+    const auto cesiumRasterOverlay = UsdUtil::getCesiumRasterOverlay(_pContext->getUsdStage(), _path);
+    int value;
+    cesiumRasterOverlay.GetMaximumSimultaneousTileLoadsAttr().Get(&value);
+    return static_cast<int>(value);
+}
+int OmniRasterOverlay::getSubTileCacheBytes() const {
+    const auto cesiumRasterOverlay = UsdUtil::getCesiumRasterOverlay(_pContext->getUsdStage(), _path);
+    int value;
+    cesiumRasterOverlay.GetSubTileCacheBytesAttr().Get(&value);
+    return static_cast<int>(value);
+}
```

Comment:
These will need to be updated to check `CesiumUtil::isSchemaValid(cesiumRasterOverlay)` once https://github.com/CesiumGS/cesium-omniverse/pull/680 is merged.

Reply:
Alright, https://github.com/CesiumGS/cesium-omniverse/pull/680 was merged

Reply:
Updated.

I'm not crazy about the required sync of default values between these accessors and the schema file:

```
int OmniRasterOverlay::getSubTileCacheBytes() const {
    const auto cesiumRasterOverlay = UsdUtil::getCesiumRasterOverlay(_pContext->getUsdStage(), _path);
    if (!UsdUtil::isSchemaValid(cesiumRasterOverlay)) {
        return 16777216;
    }
```

I don't necessarily have a better solution. I pushed the sync responsibility to a bullet in #368. It might be possible to not rely on documentation if we create a `GetDefaultAttr(prim, attrName)` function. This would look nice on the surface, but I can't think of better way than to define messy scratch prims to query. And I'm not even sure if they're immediately queryable.

Reply:
I don't like the default values either...

Maybe the default values should be grabbed from a scratch `RasterOverlayOptions` / `TilesetOptions`? This doesn't work for all files though.